django_test_migrations/migrator.py

```diff
@@ -62,10 +63,12 @@ def __init__(
         self._database: str = database
         self._executor = MigrationExecutor(connections[self._database])
 
-    def before(self, migrate_from: _Migration) -> ProjectState:
+    def before(self, migrate_from: _MigrationSpec) -> ProjectState:
         """Reverse back to the original migration."""
+        if not isinstance(migrate_from, list):
+            migrate_from = [migrate_from]
         with _mute_migrate_signals():
-            return self._executor.migrate([migrate_from])
+            return self._executor.migrate(migrate_from)
 
     def after(self, migrate_to: _Migration) -> ProjectState:
```

Comment:
Let's make `.after()` arguments compatible with `.before()`, so please update type hintings:

```suggestion
    def after(self, migrate_to: _MigrationSpec) -> ProjectState:
```

Reply:
@skarzi thanks for this catch!